slashing-protection/src/main/resources/migrations/postgresql/V6__add_db_version.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE database_version (
+    id INTEGER PRIMARY KEY,
```

Comment:
Is there even a need for a primary key? Or are planning to use to enforce we only have a single entry like with the metadata table.

Reply:
yes, it makes future updates easier.